README.rst

```diff
@@ -2,10 +2,10 @@
 matplotlib
 ##########
 
-matplotlib is a python 2D plotting library which produces publication
-quality figures in a variety of hardcopy formats and interactive
-environments across platforms. matplotlib can be used in python
-scripts, the python and ipython shell (ala matlab or mathematica), web
+matplotlib is a Python 2D plotting library which produces publication-quality
```

Comment:
Sounds familiar, and I don't remember the outcome, either.  I would say, though, that the lack of capitalization at the start of a sentence looks terrible.  Matplotlib certainly should be capitalized there, so the remaining question is whether it should be capitalized everywhere.  I think it should be.  I don't see any disadvantages.  It looks like that is the practice with Python, based on a very quick look at python.org.  Same with IPython and Jupyter.  Pandas is lower case, but italicized--that only works with styled text, so I don't consider it a good model.